org.eclipse.capra.ui.plantuml/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin
+/target
```

Comment:
Right, I did not realise that each plugin has its own .gitignore. While I feel like that is superfluous, I understand your argument to keep it consistent. If these files are automatically generated, we can just leave them in place.